stdlib/log/logger.lua

```diff
@@ -13,6 +13,8 @@ Logger = {} --luacheck: allow defined top
 -- file_extension -- a string that overides the default 'log' file extension.
 -- force_append -- each time a logger is created, it will always append, instead of
 -- -- the default behavior, which is to write out a new file, then append
+-- use_log -- whether to use log() if game.write_file isn't available, ignores other
```

Comment:
I don't like this option because it's not intuitive. It should be possible to detect control vs data.lua: see below.